tests/installation/setup_libyui_firstboot.pm

```diff
@@ -23,10 +23,27 @@ use warnings;
 use base "installbasetest";
 use registration "add_suseconnect_product";
 use version_utils "is_sle";
+use utils qw(zypper_call);
+use YuiRestClient;
+use testapi;
 
 sub run {
     add_suseconnect_product('sle-module-development-tools') if is_sle;
-    YuiRestClient::setup_libyui_firstboot();
+    zypper_call('in libyui-rest-api');
+    my $app  = YuiRestClient::get_app();
+    my $port = $app->get_port();
+    record_info('SERVER', "Used host for libyui: " . $app->get_host());
+    record_info('PORT',   "Used port for libyui: " . $port);
+    # Add the port to permanent config and restart firewalld to apply the changes immediately.
+    # This is needed, because if firewall is restarted for some reason, then the port become
+    # closed (e.g. it was faced while saving settings in yast2 lan) and further tests will not
+    # be able to communicate with YaST modules.
+    assert_script_run("firewall-cmd --zone=public --add-port=$port/tcp --permanent");
```

Comment:
we should create a function for these two lines at least I think.

Reply:
Created.